src/nrniv/nrnbbcore_write.cpp

```diff
@@ -1989,11 +2051,11 @@ int nrncore_psolve(double tstop) {
 #else // !HAVE_DLFCN_H
 
 int nrncore_run(const char*) {
-  return 0;
+  hoc_execerror("Could not launch CoreNEURON as dynamic loading not supported");
 }
 
 int nrncore_psolve(double tstop) {
-  return 0;
+  hoc_execerror("Could not launch CoreNEURON as dynamic loading not supported");
```

Comment:
Don't know that we can do the execerror at present without changing the caller. A 0 return means to go on and do the normal pc.psolve. Need to discuss a suitable set of return values for for error handling upstream

Reply:
as discussed, we will do this separately.